Specs/NumberWithUnit/Italian/CurrencyModel.json

```diff
@@ -30,12 +28,11 @@
     ]
   },
   {
-    "Input": "pagamento siegel e shuster di $94.000 dollari per ritirare tutte le richieste.",
-    "NotSupported": "dotnet",
+    "Input": "pagamento siegel e shuster di 94.000 dollari per ritirare tutte le richieste.",
     "NotSupportedByDesign": "python,javascript,python",
     "Results": [
       {
-        "Text": "$94.000",
```

Comment:
Does the form starting with $ not happen in Italian (formal or informal)?

Reply:
The test case had the unit before and after ($ and dollari). 
we modified the case in the last commit